agentos/agent_run.py

```diff
@@ -192,11 +192,14 @@ def add_episode_data(self, steps: int, reward: float):
         )
 
     def end(
-        self, status: str = RunStatus.to_string(RunStatus.FINISHED)
+        self,
+        status: str = RunStatus.to_string(RunStatus.FINISHED),
+        print_results: bool = False,
```

Comment:
Maybe this should default to `True` so we still get the little console printout when we run the acme agents?

Reply:
Done.